### ISSUE TITLE
Remove the "It's ready" notification (not the Ubuntu one)

### ISSUE THREAD

**Issue by nplezka** [alindt/WhatsUpp#12]
### System details

* OS: Ubuntu 19.04
* App Version: 0.6.1
* Desktop Environment: GNOME 3.22

### Expected behaviour
To hide the "It's ready" WhatsApp notification

[what really happened]
WhatsUpp sends a system notification "It's ready" just before the actual message notification. 

[steps to do in order to reproduce the bad behavior]
Receive a WhatsApp message when in background.

**Comment by alindt** [comment]
I'm not quite sure what this "It's ready" notification means — can you post a screenshot?

**Comment by alindt** [comment]
@nplezka: From the top of my head, this notification seems to be a feature/option of GNOME, appearing when an app sets its window to an URGENT state.
I'll have to look into it, as I don't use GNOME ­— meanwhile please check your system notification settings for something related to "URGENT" state of windows.

**Comment by nplezka** [comment]
@alindt told you, it's not the same notification... see, it's a WhatsApp notification itself. Also I've disabled the Gnome one. I can tell you I saw it already on my chinese smartband connected on my Android phone. Also, with another smartband (like a Xiaomi MiBand 2) or smartwatch there weren't problems.

**Comment by alindt** [comment]
> I can tell you I saw it already on my chinese smartband connected on my Android phone.

I don't see the connection between WhatsUpp and smartbands/android...

I've tested v0.6.1 on a fresh Ubuntu 19.04 install and I can't reproduce the notification you're talking about when receiving messages in WhatsUpp — only the message appears, no other notification.

Moreover, the same "is ready" kind of notification appeared when launching a different app that opened a window behind another app's window that was in front:

![ubuntu](https://user-images.githubusercontent.com/1379385/57480429-56551700-72a8-11e9-9a94-f514bd5b1baa.png)

So, as suspected, this issue seems to be a bug/feature/option of GNOME/Unity.

**Comment by nplezka** [comment]
I don't know how to explain this is not the same notification... that GNOME notification appears when you open an application in background, so il tells to you it's ready to be opened. WhatsUpp here is 1. already opened, 2. it's minimized. That is reproduced by WhatsApp in some way, everytime I receive a notification. It's nothing that has to do with the application itself.
I told you about the smartband because also WhatsApp for Android (with the original application, obv) showed me that notification sometimes, maybe because the smartband software wasn't optimized for WhatsApp. 
I'm telling you this because I had many WhatsApp apps built with Electron just for Linux (like Whatsie), and no one showed that notification.

**Comment by alindt** [comment]
I've tested this with WhatsUpp's window minimized/closed/etc and I just couldn't reproduce that notification.
I understand your frustration, but I can't repair something I can't reproduce.
WhatsUpp is going through some major changes in code, so maybe in the future this issue will be resolved indirectly.
You can always give the latest dev-release a try.

**Comment by nplezka** [comment]
I'm going to test the latest build.
Did you receive messages? Only in that case the notification appears.

**Comment by alindt** [comment]
Yes, of course I sent messages in order to test this — only the message notification appears, nothing else.

**Comment by nplezka** [comment]
Also the dev-release has that problem... Anyway I can live with it, but I wanted to let you know.

**Comment by alessiofiore** [comment]
I have the same problem with Skype. I think it's not related to WhatsApp, but to this version of Ubuntu

**Comment by diospiroverde** [comment]
I was able to make this disappear on Ubuntu 20.10 by commenting out the line this.win.flashFrame(true); in the Notification() function of MainBrowser.ts. 

flashFrame has no effect with my configuration (using dash-to-panel with gnome-shell panel hidden), but commenting out this line may affect other configurations by preventing the window from flashing.
